pkg/agent/bakerapi.go

```diff
@@ -130,3 +140,23 @@ func (agentBaker *agentBakerImpl) GetDistroSigImageConfig(
 
 	return allDistros, nil
 }
+
+func (agentBaker *agentBakerImpl) GetCachedVHDImages() ([]string, error) {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("error getting current working directory: %w", err)
+	}
+	manifestFilePath := filepath.Join(pwd, "../../parts/linux/cloud-init/artifacts/manifest.json")
```

Comment:
actually, if we care about container images we should be reading from components.json instead